Tests/Common/SymbolRepresentationTests.cs

```diff
@@ -34,6 +34,9 @@ public void OptionSymbolAliasMatchesOSI()
 
         [TestCase("SPXW  230111C02400000", SecurityType.IndexOption, OptionStyle.European, "SPXW", "SPX")]
         [TestCase("SPY   230111C02400000", SecurityType.Option, OptionStyle.American, "SPY", "SPY")]
+        [TestCase("AAPL240614C00100000", SecurityType.Option, OptionStyle.American, "AAPL", "AAPL")]
+        [TestCase("MSFT240614C00150000", SecurityType.Option, OptionStyle.American, "MSFT", "MSFT")]
+
```

Comment:
Nice! Thank you, let's please expand the unit test to actually assert the Option Strike, Style and Expiration date too, can add these as testcase arguments

Reply:
will do the necessary changes as you  told